tests/pipeline/test_estimate.py

```
@@ -0,0 +1,167 @@
+            print(f"❌ Error: No .ate.txt file found in {data_dir}")
+            return None
+
+        if len(ate_files) > 1:
+            print(
+                f"⚠️  Warning: Multiple .ate.txt files found, using first: {ate_files[0]}"
+            )
+
+        ate_file = ate_files[0]
+
+        # Read and parse the file
+        with open(ate_file, "r") as f:
```

Pylint (3.3.7)

[warning] Using open without explicitly specifying an encoding

(W1514)